src/core/refstrings.h

```diff
@@ -6,6 +6,8 @@
 #if GLIB_CHECK_VERSION(2, 58, 0)
 
 #define i_refstr_init() /* nothing */
+/* callback implementation */
+void i_refstr_release(char *str);
 #define i_refstr_release(str) ((str) == NULL ? NULL : g_ref_string_release(str))
```

Comment:
This sounds potentially bad? Having the name defined as both a function and a macro and needing an undef in refstrings.c

Reply:
just make all of them functions then?

Reply:
Yeah, good idea, sounds more consistent from api/abi perspective too